### PR TITLE
Templates improvements

### PR DESCRIPTION
**Pull request by mkurek**
- added new template loader, which allow to specify app label, ex. when extending template
- added additional buttons block to submit_line template
- added helper to simplify getting old object pk from new object

### CODE REVIEW

**Comment by ar4s** [comment]
:ok:

File: src/ralph/lib/template/loaders.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        return join(template_dir, template_name)
+
+    def load_template_source(self, template_name, template_dirs=None):
+        filepath = self.get_template_path(template_name, template_dirs)
+        try:
+            with io.open(filepath, encoding=self.engine.file_charset) as fp:
+                return fp.read(), filepath
+        except IOError:
+            pass
+        raise TemplateDoesNotExist(template_name)
```
Comment (ghost) [inline src/ralph/lib/template/loaders.py]: this could be instead of `pass`?

Comment (mkurek) [inline src/ralph/lib/template/loaders.py]: done

**Comment by szok** [comment]
:ok:

**Pull request merged**